pkgs/by-name/li/libserialport/package.nix

```diff
@@ -12,8 +12,8 @@ stdenv.mkDerivation rec {
   version = "0.1.2";
 
   src = fetchurl {
-    url = "https://sigrok.org/download/source/libserialport/${pname}-${version}.tar.gz";
-    sha256 = "sha256-XeuStcpywDR7B7eGhINQ3sotz9l1zmE7jg4dlHpLTKk=";
+    url = "https://sigrok.org/download/source/libserialport/libserialport-${version}.tar.gz";
+    hash = "sha256-XeuStcpywDR7B7eGhINQ3sotz9l1zmE7jg4dlHpLTKk=";
```

Comment:
Please keep commits on topic. (This has nothing to do with enabling Windows support.) Adding multiple commits in a PR is fine.

Reply:
Done